inlong-manager/manager-pojo/pom.xml

```diff
@@ -69,6 +69,18 @@
             <groupId>io.swagger</groupId>
             <artifactId>swagger-annotations</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Need to supplement the LICENSE and Notice of the third-party dependency package (if any)

Reply:
They were already added to the `licenses/inlong-manager/LICENSE`.